plugins/welcome-plugin/resources/welcome-page.css

```diff
@@ -296,23 +296,16 @@ html, body {
     margin-right: 5px;
 }
 
-.che-welcome-header .che-welcome-header-title .svg-container {
-  display: flex;
-  align-items: center;
+.che-welcome-header .che-welcome-header-title .image-container,
+.che-welcome-header .che-welcome-header-title .product-name {
+    display: flex;
+    align-items: center;
 }
 
 .che-welcome-command-desc {
     display: flex;
 }
 
-.che-welcome-command-desc .monaco-keybinding {
```

Comment:
hello, why these margins are gone ? (and for cheico logo) ?

Reply:
Style is not used

Reply:
I didn't found it in HTML, so I removed